airflow/models/variable.py

```diff
@@ -124,15 +125,18 @@ def get(
         key: str,
         default_var: Any = __NO_DEFAULT_SENTINEL,
         deserialize_json: bool = False,
+        skip_external_backends: bool = False,
```

Comment:
should we rename it to "skip_secret_backends"? I think this would be more consistent with the currently used naming  

https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/secrets-backend/index.html

For example we can have local FilesystemBackend (which is not `external`). Also it would indicate better the intentions. 

What we want to do by adding the flag we want to explicitly read values that we know are not "secret" and we do not want to look for them in secret backend.

Reply:
Also plural suggests that we can have more than one "external/secret" backend configured - which we don't have (and will not likely have), so I suggest to use singular form.

Reply:
the "problem" is that the `LocalFilesystemBackend` need to be setup as a secret_backend https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/secrets-backend/local-filesystem-secrets-backend.html

but we don't want skip it cause it's not an "external" backend 

so `skip_secret_backend` will not be very clear for people with a LocalFilesystemBackend setup

that's why originally I come up with the name `skip_external_backends`

what should we do ?

the documentation of the arg should precise the 3 not skipped backends ?